ktor-features/ktor-auth-jwt/jvm/src/io/ktor/auth/jwt/JWTAuth.kt

```diff
@@ -148,10 +149,11 @@ fun Authentication.Configuration.jwt(
     val authenticate = provider.authenticationFunction
     val verifier = provider.verifier
     val schemes = provider.schemes
+    val useForbiddenResponse = provider.useForbiddenResponse
```

Comment:
Instead of using switch options that is not composable, we usually use challenge sealed class like this one: https://github.com/ktorio/ktor/blob/9fdc5050396a9785fc39bad9cb5f8e4c0b2567c1/ktor-features/ktor-auth/jvm/src/io/ktor/auth/SessionAuth.kt#L126 
The only problem is that it is done every time for every feature and not generalized yet...